### PR TITLE
replace resources.antmedia.io with new docs url

### PR DESCRIPTION
**Pull request by timantmedia**
Replace all references to resources.antmedia.io

### CODE REVIEW

**Comment by asharamseervi** [comment]
@timantmedia Can you please resolve conflicts?

**Comment by timantmedia** [comment]
I think thats now good. But what are the failed checks? @asharamseervi

File: docusaurus.config.js [hunk 1]
```diff
@@ -68,7 +68,7 @@ const config = {
     /** @type {import('@docusaurus/preset-classic').ThemeConfig} */
     ({
       navbar: {
-        title: 'Ant Media Documentation',
```
Comment (asharamseervi) [inline docusaurus.config.js:71]: Let's not remove title ;)

File: src/components/VideoPlayer.js [hunk 2]
```diff
@@ -5,7 +5,11 @@ export default function VideoPlayer({children, video, youtube}){
 
     if(youtube == "true"){
         return (
+            <>
             <iframe className="border border-rounded m-3" width="100%" height="450" src={video} title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>
+            {children}
```
Comment (asharamseervi) [inline src/components/VideoPlayer.js:10]: These changes are already there in the master, can you please resolve this, by removing them?

**Pull request merged**